packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java

```diff
@@ -464,7 +464,7 @@ private void prepareMediaRecorder(String outputFilePath) throws IOException {
       mediaRecorder.setOutputFormat(MediaRecorder.OutputFormat.MPEG_4);
       mediaRecorder.setAudioEncoder(MediaRecorder.AudioEncoder.AAC);
       mediaRecorder.setVideoEncoder(MediaRecorder.VideoEncoder.H264);
-      mediaRecorder.setVideoEncodingBitRate(1024 * 1000);
+      mediaRecorder.setVideoEncodingBitRate(3000000);
       mediaRecorder.setAudioSamplingRate(16000);
       mediaRecorder.setVideoFrameRate(27);
       mediaRecorder.setVideoSize(videoSize.getWidth(), videoSize.getHeight());
```

Comment:
I'm wondering if using existing profiles would be a better option:
`mediaRecorder.setProfile(CamcorderProfile.get(CamcorderProfile.QUALITY_480P));` 
What do you think?

Reply:
I implemented a solution using CamcorderProfiles and matching quality between Android and iOS. Comments and suggestions are welcome.
https://github.com/flutter/plugins/pull/1403

Reply:
I think that's indeed the way to go!

Reply:
Any update in this case?